mfes/content/src/pages/content.tsx

```
@@ -22,6 +49,7 @@ interface ContentItem {
   mimeType: string;
   description: string;
   posterImage: string;
+  children: [{}];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)